src/ansys/mapdl/core/misc.py

```diff
@@ -425,3 +429,425 @@ def check_valid_start_instance(start_instance):
         )
 
     return start_instance.lower() == "true"
+
+
+def requires_update(requires_update=False):
+    def decorator(function):
+        @wraps(function)
+        def wrapper(self, *args, **kwargs):
+            if requires_update or not self._stats:
+                self._update()
+            return function(self, *args, **kwargs)
+
+        return wrapper
+
+    return decorator
+
+
+class Information:
```

Comment:
Add this information class to our API documentation as another class in https://mapdldocs.pyansys.com/api/mapdl.html

Reply:
Done.